qiita_pet/templates/study_ajax/data_type_menu.html

```diff
@@ -17,7 +17,12 @@ <h4 class="panel-title">
           <div class="panel-body">
             {% for prep in prep_info[dt] %}
               <a href="#" style="display:block;color:black;" onclick="populate_main_div('{% raw qiita_config.portal_dir %}/study/description/prep_template/', { prep_id: {{prep['id']}}, study_id: {{study_id}} });">
-                <span id="prep-header-{{prep['id']}}">{{prep['name']}} - ID {{prep['id']}} - {{prep['status']}}</span><br/>
+                <span id="prep-header-{{prep['id']}}">
+                  {% if prep['ebi_experiment'] %}
+                    <img src="{% raw qiita_config.portal_dir %}/static/img/ena.png" style="width:20px;height:20px;">
```

Comment:
Perhaps, no strong feelings. I think it's fine as it is. If we add, suggest adding to the full study as EBI doesn't have a  group preparation (preparation info file) idea.